luvit-loader.lua

```diff
@@ -207,7 +207,7 @@ local function loader(dir, path, bundleOnly)
     if package.loaded[fullPath] then
       return package.loaded[fullPath]
     end
-    local module = assert(loadfile(fullPath))()
+    local module = assert(loadfile(fullPath))(fullPath)
```

Comment:
Is there precedent for that convention being used in package.loaded? It seems like whatever key is used for package.loaded is what should be passed to the function, so if '@' is appended to this parameter then it should also be appended to the package.loaded key.

Basically, I think this should always succeed:

```
-- module.lua
return {name = ...}
```

```
local module = require('module')
assert(module == package.loaded[module.name])
```